firehose-source/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+    <artifactId>firehose-source</artifactId>
+    <name>firehose-source</name>
+
+    <parent>
+        <artifactId>spring-cloud-stream-modules</artifactId>
+        <groupId>org.springframework.cloud.stream.module</groupId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <start-class>demo.FirehoseApplication</start-class>
```

Comment:
@markfisher I've updated the project pom, still the imports profile is kicking in, and avoiding the protoc task to run. That was the rationale behind not using spring-cloud-stream as a top project, but instead a dependency. 

Any ideas on how we can solve this issue?